### PR TITLE
Updated Melissa Data descriptions and Components

### PR DESCRIPTION
**Pull request by SombreroPatrick**
Updated Melissa Data descriptions for PDI Marketplace version 6.0
Also added the profiler component to the marketplace.

### CODE REVIEW

File: marketplace.xml [hunk 1]
```diff
@@ -5430,58 +5430,70 @@ With this plugin you can:
...
-
-      IP Locator Component
-      • Identify web visitor's geographic location
-      • Append Latitude, longitude, ISP, City, State, Country
-      • Reduce fraud and geo target
-
-      Processing Options -(Local)(Cloud)(Dedicated Cloud)
-
+    <description>
+	<h1>Data Quality Components for Pentaho by Melissa Data:</h1>
```
Comment (graimundo) [inline marketplace.xml]: All html must be encapsulated by a CDATA element.

File: marketplace.xml [hunk 2]
```diff
@@ -5490,43 +5502,55 @@ With this plugin you can:
...
-    <description>Global Verify Component
-      • Address verification for over 240 countries
-      • Parses and Standardizes data based on country specific rules
-      • Transliteration of many major character sets and display output in either native or Roman characters
-      • Append Geographic coordinates for many worldwide postal addresses.
-
-      Processing Options -(Cloud Only)
-
+    <description>
+	  <h1>Global Data Quality Component for Pentaho by Melissa Data:</h1>
```
Comment (graimundo) [inline marketplace.xml]: All html must be encapsulated by a CDATA element.

File: marketplace.xml [hunk 3]
```diff
@@ -5534,48 +5558,51 @@ With this plugin you can:
...
-      standardize, correct, verify and append missing information.
-      • Move update - Identifies moves with up to a 10 year history.
-      • Append DEMOGRAPHIC data such as Date of Birth, Household Income, Occupation,
-        Length of Residence and more! (Recently Added)
-
-      Processing Options -(Cloud Only)
-
-
+    <description>
+	<h1>Personator Component for Pentaho by Melissa Data:</h1>
```
Comment (graimundo) [inline marketplace.xml]: All html must be encapsulated by a CDATA element.

File: marketplace.xml [hunk 4]
```diff
@@ -5584,15 +5611,70 @@ With this plugin you can:
...
+    <small_img>http://update2.melissadata.com/pentaho/images/PDI_MD_Profiler_s32.png</small_img>
+    <name>MDProfiler -- Data Profiler</name>
+    <category>
+      <name>Data Quality</name>
+      <parent>
+        <name>PDI Plugins</name>
+      </parent>
+    </category>
+    <description>
+	<h1>Profiler Component for Pentaho by Melissa Data:</h1>
```
Comment (graimundo) [inline marketplace.xml]: All html must be encapsulated by a CDATA element.

**Comment by graimundo** [comment]
@SombreroPatrick please encapsulate all html in a CDATA element.

**Comment by SombreroPatrick** [comment]
Ok, Added CDATA tags around the HTML descriptions. Thanks,

**Comment by graimundo** [comment]
@SombreroPatrick The pull request was a bit strange with some merges and rebases of the same commits. I went ahead and did a clean up with a cleaner pull request: https://github.com/pentaho/marketplace-metadata/pull/402
Your changes should be visible now.

**Pull request closed without merging**